torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -612,4 +614,12 @@ def functional_call(*args, **kwargs):
             )
         return out
 
+    # Note [Preserving the nn module stack metadata during export non-strict mode]
+    # This path is currently only used by the non-strict export flow,
+    # where we cannot rely on dynamo to preserve nn stack metadata in our captured graph.
+    # Instead, we stash the original user nn module here, and rely on `make_fx` to grab
+    # this stashed module and use it to track nn module stack metadata
+    if store_orig_mod:
+        functional_call._orig_mod = mod  # type: ignore[attr-defined]
```

Comment:
From zhengxu: check if _orig_mod exist?

Reply:
@tugsbayasgalan yeah as long as we add a safety check to not overwrite a preexisting attr, then I feel this might be good enough in the short term, and I can approve this.

Reply:
Updated